completions/gron.bash

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Bash Shell commandline completions for gron
+#
+# Stick this file in your ~/.bash_completion file
+#
+# You can run the command: $ mv ./gron.bash ~/.bash_completion
+
+function _gron_completion {
+  local AVAILABLE_COMMANDS="-c --colorize -j --json -k --insecure -m --monochrome -s --stream -u --ungron --no-sort --version"
```

Comment:
I'd suggest removing the short options from this list. They are unlikely to be helpful -- if one remembers a short option, one can just type it; if one does not remember, displaying a list of short options is rarely helpful, but rather adds just noise.

FWIW this is the way we do things in bash-completion, guideline from there has some more elaboration/different words, https://github.com/scop/bash-completion/blob/master/CONTRIBUTING.md?plain=1#L136-L149

Reply:
(I'd suggest moving towards https://github.com/tomnomnom/gron/issues/116 instead of manually maintaining completions separately though.)